tests/queries/0_stateless/03312_issue_74299.sql

```diff
@@ -0,0 +1,11 @@
+DROP TABLE if exists t0;
+
+CREATE TABLE t0 (c0 Int) ENGINE = Memory;
+
+INSERT INTO t0 values (1);
+
+SELECT * FROM t0 where 1;
+
+INSERT INTO TABLE t0 (c0) VALUES (INTERVAL currentProfiles() MINUTE); -- { clientError ILLEGAL_TYPE_OF_ARGUMENT }
```

Comment:
Afais, the inserted data doesn't need to be of type `INTERVAL` for the problem to reproduce. `VALUES (currentProfiles())` would be just fine.

(minor: `if exists` isn't needed in l. 11.)

(minor: why does the test need l. 5 and l. 7?)

Reply:
VALUES (currentProfiles()) works fine. I’ve fixed that.